WordPress/src/main/java/org/wordpress/android/widgets/PostListButtonType.kt

```diff
@@ -17,13 +17,19 @@ enum class PostListButtonType constructor(
     BUTTON_PREVIEW(3, R.string.button_preview, R.drawable.ic_external_white_24dp, R.color.neutral_500),
     BUTTON_STATS(4, R.string.button_stats, R.drawable.ic_stats_alt_white_24dp, R.color.neutral_500),
     BUTTON_TRASH(5, R.string.button_trash, R.drawable.ic_trash_white_24dp, R.color.neutral_500),
-    BUTTON_DELETE(6, R.string.button_delete_permanently, R.drawable.ic_trash_white_24dp, R.color.neutral_500),
+    BUTTON_DELETE(6, R.string.delete, R.drawable.ic_trash_white_24dp, R.color.neutral_500),
     BUTTON_PUBLISH(7, R.string.button_publish, R.drawable.ic_reader_white_24dp, R.color.neutral_500),
     BUTTON_SYNC(8, R.string.button_sync, R.drawable.ic_reader_white_24dp, R.color.neutral_500),
     BUTTON_MORE(9, R.string.button_more, R.drawable.ic_ellipsis_white_24dp, R.color.neutral_500),
     BUTTON_SUBMIT(10, R.string.submit_for_review, R.drawable.ic_reader_white_24dp, R.color.neutral_500),
     BUTTON_RETRY(11, R.string.button_retry, R.drawable.ic_refresh_white_24dp, R.color.error),
-    BUTTON_MOVE_TO_DRAFT(12, R.string.button_move_to_draft, R.drawable.ic_refresh_white_24dp, R.color.neutral_500);
+    BUTTON_MOVE_TO_DRAFT(12, R.string.button_move_to_draft, R.drawable.ic_refresh_white_24dp, R.color.neutral_500),
+    BUTTON_DELETE_PERMANENTLY(
```

Comment:
I think we should probably add it also [here](https://github.com/wordpress-mobile/WordPress-Android/blob/2de7ad6d3fc2f880ed884a73b3144459ef4909c4/WordPress/src/main/res/values/attrs.xml#L97) . It's currently not used directly from any layout file, but we should keep it in sync until we implement https://github.com/wordpress-mobile/WordPress-Android/issues/9533. Wdyt?

Reply:
I'm not sure what you mean (your link point to an old revision of the `attrs.xml` file). I already added it in the `wpPostButtonType` attributes: https://github.com/wordpress-mobile/WordPress-Android/pull/10185/files#diff-9135bd2ee27cc3d88e41a5881ebd2dee

Reply:
ohh, my bad sorry :X. I'd swear the change wasn't there yesterday 😕